SCClassLibrary/Common/Control/Server.sc

```diff
@@ -265,8 +265,9 @@ Server {
 	classvar <>named, <>all, <>program, <>sync_s = true;
 	classvar <>nodeAllocClass, <>bufferAllocClass, <>busAllocClass;
 
-	var <name, <addr, <clientID, <userSpecifiedClientID = false;
```

Comment:
are we OK to delete this public instance member?

Reply:
It was only used internally to determine whether a clientID was specified when creating a server - otherwise, clientID was not considered wise to change. 
Now, clientID can cleanly be set by users while the server is off, and the user give ID will be requested on the next boot. 
So I think this is fine to remove. I cannot think of a case where user code would be interested in userSpecifiedClientID. 
Do you think it would be better to post an explanation, and fail? or return false? 
```
+ Server { 
	userSpecifiedClientID { 
		this.deprecated(thisMethod);
		"Server:clientID can now be set while a server is off, and is locked while the server is running. Thus, userSpecifiedClientID is no longer needed internally, and meaningless. Returns false just in case user code calls this.".postln;
		^false
	}
}
```

Reply:
It would we should deprecate it. If it is in the public interface, we need to give some warning, even if it seems like there's no obvious use case.

Reply:
done.